.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+# 2.1 does not yet support local run
+# unless with workaround. For simplicity just use it.
+# https://github.com/CircleCI-Public/circleci-cli/issues/79
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8
+    steps:
+      - checkout
+      - run:
+          name: Update npm
+          command: 'sudo npm install -g npm@latest'
```

Comment:
Why update to latest instead of using the default one?

Reply:
That was a standard setup widely suggested by CircleCI, I considered keeping them because the part that includes sloc_nightly, but since we are not introducing sloc_nightly right now, I think we can remove the update `npm`

Reply:
DONE.